packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -35,6 +44,30 @@ export default function ViewApplicationPage() {
     })
   })
 
+  const credentials: ProjectCredentials = application?.project!.credentials ?? {}
+
+  useEffect(() => {
+    if (!credentials) {
+      return
+    }
+    const verify = async () => {
+      const newVerifiedProviders: { [key: string]: VerifiedCredentialState } = { ...verifiedProviders }
+      for (const provider of Object.keys(verifiedProviders)) {
+        const verifiableCredential = credentials[provider]
+        if (!!verifiableCredential) {
+          newVerifiedProviders[provider] = await verifier.verifyCredential(verifiableCredential)
+            ? VerifiedCredentialState.VALID
+            : VerifiedCredentialState.INVALID
+        }
+      }
+
+      setVerifiedProviders(newVerifiedProviders)
+    }
+
+    verify()
+  }, [])
```

Comment:
need to add a ts-ignore for `Line 68:6:  React Hook useEffect has missing dependencies: 'credentials', 'verifiedProviders', and 'verifier'. Either include them or remove the dependency array  react-hooks/exhaustive-deps` warning